File: prepper-bunker-backend/src/main/java/org/jonasfroeller/models/Drink.java
package org.jonasfroeller.models;

import jakarta.persistence.*;
import io.quarkus.hibernate.orm.panache.PanacheEntity;
import java.time.LocalDate;

@Entity
public class Drink extends PanacheEntity {
    public String type;
    public double quantity;
    public LocalDate expirationDate;

    @ManyToOne
    @JoinColumn(name = "storage_location_id")
    public StorageLocation storageLocation;
}

File: prepper-bunker-frontend/src/app/models/drink.ts
export interface Drink {
  id: number;
  type: string;
  quantity: number;
  expirationDate: string;
  storageLocationId: number;
}

export interface DrinkCreate {
  type: string;
  quantity: number;
  expirationDate: string;
  storageLocationId: number;
}
